posthog/api/test/test_feature_flag.py

```diff
@@ -1033,10 +1033,7 @@ def test_local_evaluation(self, mock_capture):
                 },
                 "deleted": False,
                 "active": True,
-                "is_simple_flag": True,
-                "rollout_percentage": 20,
                 "ensure_experience_continuity": False,
-                "experiment_set": [],
```

Comment:
The frontend is using `experiment_set` to determine if a feature flag is linked to an experiment, how much faster does removing this make the local eval?

Reply:
This endpoint isn't called by the frontend, it's for client libraries, so nothing needs to change for the frontend :)

Reply:
We can probably just make an extra call to experiments in the FE and filter for those feature flags if we have to

Reply:
And 20-50% depending on the team, but the key is that this endpoint is called every 30 seconds by every client library out in the wild, so even small gains add up

Reply:
These tests are simulating what's received by the client libraries

Reply:
i see! very nice